### PR TITLE
Update readme: Add instructions for installation via config, improve documentation

### PR DESCRIPTION
**Pull request by JohnRTitor**
- Clearly state which packages are available
- Add instructions for installation via config
- Add flake description
- Improve titles

Overall, improve documentation of this great effort. Thanks :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,18 +1,62 @@
...
 
-google-chrome-beta and google-chrome-dev were not getting maintained in nixpkgs,
-and were dropped in https://github.com/NixOS/nixpkgs/pull/261870.
+This flake provides the latest Chrome stable, beta and dev releases. The flake is updated regularly to ensure fast and up to date releases, which would otherwise not be possible in [nixpkgs](https://github.com/NixOS/nixpkgs).
 
+## Why?
 
-# how?
+`google-chrome-beta` and `google-chrome-dev` were not getting maintained in nixpkgs,
+and were dropped in [NixOS/nixpkgs#261870](https://github.com/NixOS/nixpkgs/pull/261870).
```
Comment (r-k-b) [inline README.md:8]: thought: the extra markdown isn't necessary, it already displays with the short name when rendered:

image

**Review by r-k-b — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (JohnRTitor) [reply README.md:8]: Oh... I was getting a warning on my markdown linter that this is a naked link, so I added the label.

File: README.md [hunk 2]
```diff
@@ -1,18 +1,62 @@
...
 
+## Available packages
+
+- `google-chrome`
```
[the remark this answers is not in the document]
Comment (JohnRTitor) [reply README.md]: Removed the list and added a note about `nix flake show`.

[the remark this answers is not in the document]
Comment (r-k-b) [reply README.md]: Thanks!

**Pull request merged**